### PR TITLE
Add an interactive  photo gallery.

### PR DESCRIPTION
**Pull request by andreeanica16**
Add more content to the introduction page and make it more user-friendly.

### CODE REVIEW

**Comment by andreeanica16** [comment]
@iht  @juliocc

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,71 +12,139 @@
...
+    Creates the collection of elements and store them in 
+    allGalleryElements
+ */
+ let allGalleryELements = [];
+ for (let i = 0; i < text.length; i++) {
+     let elem = new GalleryElement('images/img' + i + '.jpg', text[i]);
+     allGalleryELements.push(elem);
+ }
+
+//Memorize the index of the first picture to be displayed
```
Comment (iht) [inline portfolio/src/main/webapp/script.js]: Space

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,71 +12,139 @@
...
+                'My beautiful homeland',
+                'Travelling in my country',
+                'My escape from the city',
+                'This summer I would have visited Florence',
+                'I am passionate about art',
+                ];
+ /*
+    Creates the collection of elements and store them in 
+    allGalleryElements
+ */
```
Comment (iht) [inline portfolio/src/main/webapp/script.js]: Different comment style to the other functions

**Review by iht — APPROVED** [review]
LGTM

**Pull request merged**